src/components/Label/Label.js

```diff
@@ -20,11 +21,13 @@ const baseStyles = ({ theme }) => css`
 /**
  * Label component for forms.
  */
-const Label = styled('label')`
+const StyledLabel = styled('label')`
```

Comment:
Hm, why is this necessary? Looks to me like `Label` and `StyledLabel` are identical. Can we revert this?